ts/client/src/accounts/group.ts

```diff
@@ -472,11 +470,23 @@ export class Group {
       uiPrice = this.toUiPrice(price, baseDecimals);
       lastUpdatedSlot = stubOracle.lastUpdateSlot.toNumber();
       provider = OracleProvider.Stub;
+      deviation = stubOracle.deviation;
     } else if (isPythOracle(ai)) {
       const priceData = parsePriceData(ai.data);
       uiPrice = priceData.previousPrice;
       price = this.toNativePrice(uiPrice, baseDecimals);
       lastUpdatedSlot = parseInt(priceData.lastSlot.toString());
+      deviation = priceData.confidence
+        ? I80F48.fromNumber(
```

Comment:
I think you want to deal with `undefined` but for confidence == 0 we should set deviation to 0 too. So `confidence !== undefined`?